logging_alerts/custom-logging-alerts.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In logging 5.7 and later versions, users can configure the LokiStack deployment to produce customized alerts and recorded metrics. If you want to use customized link:https://grafana.com/docs/loki/latest/alert/[alerting and recording rules], you must enable the LokiStack ruler component.
+You can configure the LokiStack deployment to produce customized alerts and recorded metrics. If you want to use customized link:https://grafana.com/docs/loki/latest/alert/[alerting and recording rules], you must enable the LokiStack ruler component.
```

Comment:
I was recently made aware of the fact that while we provide an easy way to make the alerts visible in the console, there currently seems to be no easy way to get the values of the recording rules into the default Prometheus, because it needs a remote-write endpoint, which we are apparently lacking.

It's possible to configure the remote-write in LokiStack (using a RulerConfig), but we're not setting this up by default currently, which renders RecordingRules a lot less useful than AlertingRules.

I wonder if we should mention the fact that RecordingRules need additional setup to work properly somewhere.

Reply:
Do you think this can be done in a follow-up Jira?